oml/lightning/entrypoints/parser.py

```diff
@@ -49,4 +60,104 @@ def check_is_config_for_ddp(cfg: TCfg) -> bool:
+        )
+        dict_to_log = {**dictconfig_to_dict(cfg), **{"dir": cwd}}
+        logger.log_hyperparams(flatten_dict(dict_to_log, sep="|"))
+        upload_files_to_neptune_cloud(logger, cfg)
+
+    else:
+        print(f"In order to use files, you should use {NeptuneLogger.__name__}")
+        logger = True
+
+    return logger
+
+
+def upload_files_to_neptune_cloud(logger: NeptuneLogger, cfg: TCfg) -> None:
+    assert isinstance(logger, NeptuneLogger)
+
+    # save transforms as files
+    for key, val in cfg.items():
+        if "transforms" in key:
+            try:
+                transforms = get_transforms_by_cfg(cfg[key])
+                if isinstance(transforms, albu.Compose):
+                    transforms_file = str(Path(".hydra/") / f"{key}.yaml") if Path(".hydra").exists() else f"{key}.yaml"
+                    albu.save(filepath=transforms_file, transform=transforms, data_format="yaml")
+                    logger.run[key].upload(str(transforms_file))
+            except Exception:
+                print(f"We are not able to interpret {key} as albumentations transforms and log them as a file.")
+
+    # log source code
+    source_files = list(map(lambda x: str(x), PROJECT_ROOT.glob("**/*.py"))) + list(
+        map(lambda x: str(x), PROJECT_ROOT.glob("**/*.yaml"))
+    )
+    logger.run["code"].upload_files(source_files)
+
+    # log dataset
+    logger.run["dataset"].upload(str(Path(cfg["dataset_root"]) / cfg["dataframe_name"]))
+
+
+def parse_scheduler_from_config(cfg: TCfg, optimizer: torch.optim.Optimizer) -> Dict[str, Any]:
+    if cfg.get("scheduling"):
+        scheduler_kwargs = {
+            "scheduler": get_scheduler_by_cfg(cfg["scheduling"]["scheduler"], **{"optimizer": optimizer}),
+            "scheduler_interval": cfg["scheduling"]["scheduler_interval"],
+            "scheduler_frequency": cfg["scheduling"]["scheduler_frequency"],
+            "scheduler_monitor_metric": cfg["scheduling"].get("monitor_metric", None),
+        }
+    else:
+        scheduler_kwargs = {"scheduler": None}
+
+    return scheduler_kwargs
+
+
+def parse_sampler_from_config(cfg: TCfg, dataset: DatasetWithLabels) -> IBatchSampler:
+    if (not dataset.categories_key) and cfg["sampler"]["name"] in SAMPLERS_CATEGORIES_BASED.keys():
+        raise ValueError(
+            "NOTE! You are trying to use Sampler which works with the information related"
+            "to categories, but there is no <categories_key> in your Dataset."
+        )
+
+    sampler_runtime_args = {"labels": dataset.get_labels(), "label2category": dataset.get_label2category()}
+    sampler = get_sampler_by_cfg(cfg["sampler"], **sampler_runtime_args) if cfg["sampler"] is not None else None
+
+    return sampler
+
+
+def parse_ckpt_callback_from_config(cfg: TCfg) -> ModelCheckpoint:
+    return ModelCheckpoint(
+        dirpath=Path.cwd() / "checkpoints",
+        monitor=cfg.get("metric_for_checkpointing", f"{OVERALL_CATEGORIES_KEY}/cmc/1"),
```

Comment:
is it good default? For benchmarking it's ok, but in practice, no one should be interest this metric))